beebop/app.py

```diff
@@ -750,40 +737,34 @@ def get_project(p_hash) -> json:
     """
     job_id = redis.hget("beebop:hash:job:assign", p_hash)
     if job_id is None:
-        return jsonify(error=response_failure({
-            "error": "Project hash not found",
-            "detail": "Project hash does not have an associated job"
-        })), 404
+        raise NotFound("Project hash does not have an associated job")
 
     status = get_status_internal(p_hash, redis)
 
-    if "error" in status:
-        return jsonify(error=response_failure(status)), 500
-    else:
-        clusters_result = get_cluster_assignments(p_hash, storage_location)
-        failed_samples = get_failed_samples_internal(p_hash, storage_location)
-
-        fs = PoppunkFileStore(storage_location)
-        passed_samples = {}
-        for value in clusters_result.values():
-            sample_hash = value["hash"]
-            sketch = fs.input.get(sample_hash)
-            passed_samples[sample_hash] = {
-                "hash": sample_hash,
-                "sketch": sketch,
+    clusters_result = get_cluster_assignments(p_hash, storage_location)
```

Comment:
So this isn't checking for error in the result of get_status_internal any more. Is it expecting an error to have been raised in that case?